Makefile

```diff
@@ -0,0 +1,25 @@
+include .env
+export $(shell sed 's/=.*//' .env)
+
+up:
+	docker-compose pull
+	docker-compose up -d --build --remove-orphans
+
+open:
+	open "http://$(PROJECT_BASE_URL):${PROJECT_PORT}"
+
+build:
+	docker exec -t ${PROJECT_NAME}_app sh -c 'if [ ! -e "config.php" ]; then cp config.sample.php config.php; fi'
+	docker exec -t ${PROJECT_NAME}_app sh -c 'composer install'
+
+stop:
+	docker-compose stop
+
+stop-all-containers:
+	ids=$$(docker ps -a -q) && if [ "$${ids}" != "" ]; then docker stop $${ids}; fi
+
+in:
+	docker exec -it ${PROJECT_NAME}_app bash
+
+crawl:
+	docker exec -t ${PROJECT_NAME}_app bash -c 'php bin/visual_regression_bot.php -v bot:crawl-sites sample-sites.csv'
```

Comment:
ideally we should allow at least the csv as parameter (sample-sites.csv) but in the future we may have more params, like a tag to categorise the crawl for example.

otherwise, this is super cool, thanks a lot for taking the effort @marcelovani

Reply:
Yes, I was thinking the same, but this is just a starting poing